### ISSUE TITLE
$SOLID_GLOBAL_config not defined - databrowser/mashlib won't load

### ISSUE THREAD

**Issue by tomasklapka** [solid/node-solid-server#583]
I'm getting `$SOLID_GLOBAL_config not defined` console error at solid-ui's [signin.js#L20](https://github.com/linkeddata/solid-ui/blob/269344e20bc310cef1bd3d971c13fbacfc34131b/src/signin.js#L20)
I can find this variable only in mashlib's [test/index.html](https://github.com/linkeddata/mashlib/blob/7b6aa6fd95606987a97081362b7fda7b7bff849b/test/index.html)

Mashlib loads if I define the variable in `static/databrowser.html`:
```
  <script>var $SOLID_GLOBAL_config = {}</script>
  <script type="text/javascript" src="https://linkeddata.github.io/mashlib/dist/mashlib-prealpha.js"></script>
```

**Comment by RubenVerborgh** [comment]
Yes, this code just needs to be updated to what we have on timbl.com. Also, `mashlib-prealpha` has been changed since.

**Comment by RubenVerborgh** [comment]
Example: https://github.com/linkeddata/mashlib/blob/master/test/index.html

**Comment by RubenVerborgh** [comment]
Depends on #590.

**Comment by RubenVerborgh** [comment]
Implemented in #593.
